protocol-tests/src/main/java/fixtures/paging/implementation/PagingsImpl.java

```diff
@@ -0,0 +1,3477 @@
+                @HostParam("$host") String host,
+                @HeaderParam("Accept") String accept,
+                RequestOptions requestOptions,
+                Context context);
+    }
+
+    /**
+     * A paging operation that must return result of the default 'value' node.
+     *
+     * <p><strong>Response Body Schema</strong>
+     *
+     * <pre>{@code
+     * BinaryData
+     * }</pre>
+     *
+     * @return a DynamicRequest where customizations can be made before sent to the service.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<PagedResponse<BinaryData>> getNoItemNamePagesSinglePageAsync(RequestOptions requestOptions) {
+        final String accept = "application/json";
+        return FluxUtil.withContext(
+                        context -> service.getNoItemNamePages(this.client.getHost(), accept, requestOptions, context))
+                .map(
+                        res ->
+                                new PagedResponseBase<>(
+                                        res.getRequest(),
+                                        res.getStatusCode(),
+                                        res.getHeaders(),
+                                        getValues(res.getValue(), "value"),
+                                        getNextLink(res.getValue(), "nextLink"),
+                                        null));
+    }
+
+    /**
+     * A paging operation that must return result of the default 'value' node.
+     *
+     * <p><strong>Response Body Schema</strong>
+     *
+     * <pre>{@code
+     * BinaryData
+     * }</pre>
+     *
+     * @return a DynamicRequest where customizations can be made before sent to the service.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<PagedResponse<BinaryData>> getNoItemNamePagesSinglePageAsync(
+            RequestOptions requestOptions, Context context) {
+        final String accept = "application/json";
+        return service.getNoItemNamePages(this.client.getHost(), accept, requestOptions, context)
+                .map(
+                        res ->
+                                new PagedResponseBase<>(
+                                        res.getRequest(),
+                                        res.getStatusCode(),
+                                        res.getHeaders(),
+                                        getValues(res.getValue(), "value"),
+                                        getNextLink(res.getValue(), "nextLink"),
+                                        null));
+    }
+
+    /**
+     * A paging operation that must return result of the default 'value' node.
+     *
+     * <p><strong>Response Body Schema</strong>
+     *
+     * <pre>{@code
+     * BinaryData
+     * }</pre>
+     *
+     * @return a DynamicRequest where customizations can be made before sent to the service.
+     */
+    @ServiceMethod(returns = ReturnType.COLLECTION)
```

Comment:
Not sure whether it should be `ReturnType.COLLECTION`

Reply:
I copy them from high level code.
`Mono` -> `@ServiceMethod(returns = ReturnType.SINGLE)`. 
`PagedFlux` -> `@ServiceMethod(returns = ReturnType.COLLECTION)`
`PagedIterable` -> `@ServiceMethod(returns = ReturnType.COLLECTION)`